resources/js/Pages/Gallery.vue

```diff
@@ -4,11 +4,14 @@
     <Head title="Gallery" />
     <section>
 
+
+
         <!-- Column for title and add photos button -->
         <div class="columns is-vcentered">
             <div class="column">
                 <h1 class="title is-1">Gallery view</h1>
             </div>
+            <a class="button is-primary" @click="openUpdateModal" v-if="user.id == album.id_user"><strong> Modify Album </strong></a>
```

Comment:
```suggestion
            <b-button type="is-info" v-if="user.id == album.id_user" label="Modify Album" icon-left="pencil-square-o" icon-pack="fa" @click="openUpdateModal" />
```
Better button with icon